Clarify the computation of ExtLen. (#17)
scionassociation/scion-dp_I-D: draft-dekater-scion-dataplane.md

- `NextHdr`: Unsigned 8-bit integer. Identifies the type of header immediately following the Hop-by-Hop/End-to-End Options header. Values of this field respect the Assigned SCION Protocol Numbers (see also [](#protnum)).

> - `ExtLen`: Unsigned 8-bit integer. The value of this field is computed as the length of the header in 4-octet units not including the first 4 octets. That is: `ExtLen = uint8(((options_len + 2) / 4) - 1)`, where `options_len + 2` is a multiple of 4.

Reviewer: ```suggestion
- `ExtLen`: Unsigned 8-bit integer. Length of the  Hop-by-Hop or End-to-End Options header in 4-octet units, not including the first 4 octets. That is: `ExtLen = uint8(((options_len + 2) / 4) - 1)`, where `options_len + 2` is a multiple of 4.
```

I'd try language similar to IPv6 (RFC2460): 
```   
Hdr Ext Len    8-bit unsigned integer.  Length of the Routing
                   header in 8-octet units, not including the first
                   8 octets.
```

I don't really understand the formula, and I'd not use `options_len` in there as it seems we are introducing a new field. I'd rather use tomething lile `L` and then on the side say what `L` is (extension header size in bits)

Author: I did use the language of RFC8200 for inspiration... rewrote to be even closer. The formula had to be added in order to answer the question that the reviewer asked: "how to compute it".

> - `Options`: This is a variable-length field. The length of this field MUST be such that the complete length of the Hop-by-Hop/End-to-End Options header is an integer multiple of 4 bytes. This can be achieved by using padding-only options. The `Options` field contains one or more Type-Length-Value (TLV) encoded options. For details, see [](#optfld).

Reviewer: What is a `padding-only` option?  In the draft we describe `Pad1` and `PadN`

Author: Rewrote to remove the phrase "padding-only" options and link to table-4.
